### ISSUE TITLE
rockstor-bootstrap.service fails to start on rockstor-3.8-9.06.x86_64

### ISSUE THREAD

**Issue by roweryan** [rockstor/rockstor-core#1026]
````
# systemctl |grep fail
rockstor-bootstrap.service                                                                                 loaded failed     failed          Rockstor bootstrapping tasks
# rpm -q rockstor
rockstor-3.8-9.06.x86_64
# systemctl status rockstor-bootstrap -l
rockstor-bootstrap.service - Rockstor bootstrapping tasks
   Loaded: loaded (/etc/systemd/system/rockstor-bootstrap.service; enabled)
   Active: failed (Result: exit-code) since Fri 2015-11-27 14:38:32 CET; 1min 53s ago
  Process: 2321 ExecStart=/opt/rockstor/bin/bootstrap (code=exited, status=1/FAILURE)
 Main PID: 2321 (code=exited, status=1/FAILURE)
   CGroup: /system.slice/rockstor-bootstrap.service

Nov 27 14:38:32 rockstor bootstrap[2321]: File "/opt/rockstor/src/rockstor/scripts/bootstrap.py", line 43, in main
Nov 27 14:38:32 rockstor bootstrap[2321]: aw.api_call('network')
Nov 27 14:38:32 rockstor bootstrap[2321]: File "/opt/rockstor/src/rockstor/cli/api_wrapper.py", line 85, in api_call
Nov 27 14:38:32 rockstor bootstrap[2321]: self.set_token()
Nov 27 14:38:32 rockstor bootstrap[2321]: File "/opt/rockstor/src/rockstor/cli/api_wrapper.py", line 79, in set_token
Nov 27 14:38:32 rockstor bootstrap[2321]: raise Exception(msg)
Nov 27 14:38:32 rockstor bootstrap[2321]: Exception: Exception while setting access_token for url(https://1.2.3.4): HTTPSConnectionPool(host='1.2.3.4', port=443): Max retries exceeded with url: /o/token/ (Caused by <class 'socket.error'>: [Errno 111] Connection refused). content: None
Nov 27 14:38:32 rockstor systemd[1]: rockstor-bootstrap.service: main process exited, code=exited, status=1/FAILURE
Nov 27 14:38:32 rockstor systemd[1]: Failed to start Rockstor bootstrapping tasks.
Nov 27 14:38:32 rockstor systemd[1]: Unit rockstor-bootstrap.service entered failed state.
````

Restarting it after boot works.

**Comment by schakrava** [comment]
Thanks for opening this issue @roweryan. This is now fixed by #1032

**Comment by schakrava** [comment]
Thanks for opening this issue @roweryan. This is now fixed by #1032
